src/libraries/Common/src/System/Security/Cryptography/SlhDsa.cs

```diff
@@ -394,6 +395,7 @@ public byte[] ExportEncryptedPkcs8PrivateKey(ReadOnlySpan<char> password, PbePar
         public byte[] ExportEncryptedPkcs8PrivateKey(ReadOnlySpan<byte> passwordBytes, PbeParameters pbeParameters)
         {
             ThrowIfDisposed();
+            ArgumentNullException.ThrowIfNull(pbeParameters);
```

Comment:
When SlhDsa is added to Microsoft.Bcl.Cryptography you'll find that these helpers aren't callable.  It's probably reasonable to go ahead and include SlhDsa into Microsoft.Bcl.Cryptography now (and the tests into M.B.C.Tests) so this doesn't have to be reworked after.

I think @vcsjones went with just writing out the null checks and throws, we could also make our own helper somewhere (in a helpers class already included in both projects, or once per type that uses it, or whatever):

```C#
internal static void ThrowIfNull(object arg, [CallerArgumentExpression(nameof(arg))] string? paramName = null)
{
#if NET8_OR_GREATER
    ArgumentNullException.ThrowIfNull(arg, paramName);
#else
    if (arg is null)
    {
        throw new ArgumentNullException(paramName);
    }
}
```

Reply:
The way the rest of the BCL does this is to use `ThrowHelper`. We should just use that everywhere.

Reply:
Hm, actually I am not sure I like it :-)